client/src/main/java/org/eclipse/hono/client/Command.java

```diff
@@ -265,14 +288,14 @@ public String getContentType() {
     }
 
     /**
-     * Gets this command's reply-to-id.
+     * Gets this command's original reply-to-id, as extracted from the message given to the <em>from</em> factory method.
      *
      * @return The identifier.
      * @throws IllegalStateException if this command is invalid.
      */
-    public String getReplyToId() {
+    public String getOriginalReplyToId() {
```

Comment:
IMHO we should not change the name of this method. FMPOV it is sufficient to document that the original, unaltered *reply-to* value will be returned. I guess that is the behavior that clients expect anyways.
Otherwise we would also need to document this API change in the release notes.

Reply:
Ok, yes. I've reverted this name change and adapted the javadoc instead. Also added some further javadoc emphasizing the difference between reply-to as received from the application and the adapted reply-to used when forwarding the command to the device.